src/utils/fxa.ts

```diff
@@ -155,11 +156,11 @@ async function refreshOAuthTokens(
 
     const responseJson = await response.json();
     if (!response.ok) throw new Error(responseJson);
-    console.info("refresh_fxa_access_token_success");
+    logger.info("refresh_fxa_access_token_success");
     return responseJson as FxaPostOauthTokenResponseSuccessRefreshToken;
   } catch (e) {
     if (e instanceof Error) {
-      console.error("refresh_fxa_access_token", { stack: e.stack });
+      logger.error("refresh_fxa_access_token", { stack: e.stack });
```

Comment:
From reading the GCP docs recently, we should also include `message: e.message`:

https://cloud.google.com/logging/docs/structured-logging
> If your log entry contains an exception stack trace, the exception stack trace should be set in this message JSON log field, so that the exception stack trace can be parsed and saved to Error Reporting.